scripts/end-to-end/create_traces.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I think this file is not really needed and it makes too many assumptions about paths - the instructions in the verify script are sufficient.